namespaces/live.cloud-platform.service.justice.gov.uk/jogapp-dev/01-rbac.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: jogapp-dev-admin
+  namespace: jogapp-dev
+subjects:
+  - kind: Group
+    name: "github:jo-g-app"
```

Comment:
Hi @jogilder , this should be a github team which you are part of.

Reply:
Thanks, will update it